### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by o1pranay**
This PR:
- moves some of the /docs files into /frontend/website/docs, so the edit buttons don't go to the symlinked versions
- adds FAQ, testnet landing page, and codebase overview
- other fixes to content / old docs

### CODE REVIEW

File: frontend/website/docs/architecture/lifecycle-payment-technical.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+    -privkey-path keys/my-wallet
+```
+
+Where:
+- `amount` is the amount of coda you are sending
+- `receiver` is the public key of the intended recipient
+- `fee` is the fee to be paid to the network to process the transaction
+- `privkey-path` is the filepath to the private key for your account
+
+<a name="client"></a>
```
Comment (figitaki) [inline frontend/website/docs/architecture/lifecycle-payment-technical.md]: did you manually add these anchor tags?

File: frontend/website/docs/faq.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+# FAQ
+
+## Testnet
+
+---
+
+#### Do I need any specific hardware?
+
+Currently, you can use a CPU, and we recommend at least a 4-core processor and 8 GB's of RAM.  In the near future though, we may require the use of GPU's
```
Comment (Schmavery) [inline frontend/website/docs/faq.md]:
```suggestion
Currently, we recommend at least a 4-core processor and 8 GB's of RAM.  In the near future though, we may require the use of GPUs.
```

**Comment by bkase** [comment]
Can you include a `mkdocs-yaml` change that makes PRs against the `develop` branch here?

File: frontend/website/docs/architecture/lifecycle-payment-technical.md (hunk 1 above)
Comment (o1pranay) [reply frontend/website/docs/architecture/lifecycle-payment-technical.md]: no, this is leftover from the previous doc - I'll go ahead and remove it

**Comment by Schmavery** [comment]
There are some dead links on the consensus page (you can see them in the mk-docs warnings when you build)
```
WARNING -  Documentation file 'architecture/consensus.md' contains a link to 'architecture/res/consensus_hooks.dot.png' which is not found in the documentation files. 
WARNING -  Documentation file 'architecture/consensus.md' contains a link to 'architecture/proposer.md' which is not found in the documentation files. 
```

possible that they were already there.

**Comment by o1pranay** [comment]
@Schmavery I commented out those files for now - but will add them later (need to move the dot files over to our docs repo & also change the proposer language so i didn't wanna move the proposer doc over).

**Comment by Schmavery** [comment]
@o1pranay not sure if you deployed this change but the dead links seem to still be there

**Comment by o1pranay** [comment]
@Schmavery ugh ok, I'll take a look

**Pull request merged**